tests/pipeline/resources/streams-bootstrap/defaults.yaml

```diff
@@ -1,3 +1,8 @@
+cleaner:
```

Comment:
I don't see where you validate that in tests

Reply:
enrichment from defaults does work now as long as the parent `diff_config.ignore` is not set.